src/corell/examples/Cargo.toml

```diff
@@ -0,0 +1,67 @@
+[package]
+name = "gfx-corell-examples"
```

Comment:
I wasn't originally able to make it work using one Cargo.toml file. I could probably figure it out now that I've been working on this so much, but right now it keeps the dependencies separated. I'm not saying that's a good reason to keep it this way... I'm just not super interested in iterating again today.

If you want to block on this, I'm happy to pick it up again tomorrow and see if I can figure it out. Up to you